packages/client/wallets/web3auth-adapter/src/networks.ts

```diff
@@ -0,0 +1,133 @@
+import { EVMBlockchainIncludingTestnet } from "@crossmint/common-sdk-base";
+
+export function getUrlProviderByBlockchain(chain: EVMBlockchainIncludingTestnet) {
+    const url = new Map<EVMBlockchainIncludingTestnet, string | null>([
+        ["ethereum", "https://eth.llamarpc.com"],
+        ["polygon", "https://rpc.ankr.com/polygon"],
+        ["bsc", "https://binance.llamarpc.com"],
+        ["optimism", "https://rpc.ankr.com/optimism"],
+        ["arbitrum", "https://rpc.ankr.com/arbitrum"],
+        ["ethereum-sepolia", "https://ethereum-sepolia.publicnode.com"],
+        ["polygon-amoy", "https://rpc-amoy.polygon.technology"],
+        ["zkatana", "https://rpc.startale.com/zkatana"],
+        ["zkyoto", "https://rpc.startale.com/zkyoto"],
+        ["arbitrum-sepolia", "https://sepolia-rollup.arbitrum.io/rpc"],
+        ["base-goerli", null],
+        ["base-sepolia", "https://base-sepolia-rpc.publicnode.com"],
+        ["bsc-testnet", null],
+        ["ethereum-goerli", "https://ethereum-goerli.publicnode.com"],
+        ["optimism-goerli", null],
+        ["optimism-sepolia", "https://sepolia.optimism.io"],
+        ["zora-goerli", null],
+        ["zora-sepolia", null],
+        ["base", "https://mainnet.base.org"],
+        ["zora", null],
+        ["arbitrumnova", "https://arbitrum-nova-rpc.publicnode.com"],
+        ["astar-zkevm", "https://rpc.startale.com/astar-zkevm"],
+        ["apex", null],
+    ]).get(chain)!;
+
+    if (url == null) {
+        throw new Error(`Url provider not found for chain ${chain}`);
+    }
+    return url;
+}
+
+export function getBlockExplorerByBlockchain(chain: EVMBlockchainIncludingTestnet) {
```

Comment:
Why are these part of web3auth? seems like a generic utility

Reply:
actually this applies to most of the file

Reply:
All the code in this file was exclusively used for creating the web3auth signer!

Reply:
It may be worth moving later to a core package so its usable by other parts of the SDK